brainiak/reprsimil/brsa.py

```diff
@@ -248,7 +251,8 @@ def fit(self, X, design, scan_onsets=None, coords=None,
         assert_all_finite(design)
         assert design.ndim == 2,\
             'The design matrix should be 2 dimension ndarray'
-        assert np.all(np.std(design, axis=0) >= 0) or self.pad_DC,\
+        assert (not np.all(np.std(design, axis=0) > 0) and self.pad_DC)\
```

Comment:
Thanks @TuKo What is the advantage of using raise instead of assert?
It increases complexity. I used to change all the raise to assert for the complexity tests.